tests/ci/install_environment.sh

```diff
@@ -1,29 +1,31 @@
 #!/usr/bin/env bash
 
-export NUOVERSION=4.0.7.1
+export NUOVERSION=5.0.3.3
+export TAR_VERSION=nuodb-${NUOVERSION}.linux.x86_64
 export INFLUX_VERSION=1.4.3
-
-sudo sh -c 'echo madvise > /sys/kernel/mm/transparent_hugepage/enabled'
-sudo sh -c 'echo madvise > /sys/kernel/mm/transparent_hugepage/defrag'
-wget -q http://ce-downloads.nuohub.org/nuodb-ce_${NUOVERSION}_amd64.deb --output-document=/var/tmp/nuodb.deb
-sudo dpkg -i /var/tmp/nuodb.deb
+echo "http://ce-downlaods.nuohub.org/${TAR_VERSION}.tar.gz"
+wget http://ce-downloads.nuohub.org/${TAR_VERSION}.tar.gz --output-document=/var/tmp/nuodb.tgz
+tar xzf /var/tmp/nuodb.tgz
+export NUODB_HOME=${PWD}/${TAR_VERSION}
 sleep 5
 
-export PATH=$PATH:/opt/nuodb/bin
-sudo sed -i -e "/ssl.:/s/true/false/" /etc/nuodb/nuoadmin.conf
-sudo service nuoadmin start
-nuocmd create archive --db-name hockey --server-id nuoadmin-0 --archive-path /var/opt/nuodb/demo-archives/hockey
+export PATH=$PATH:${NUODB_HOME}/bin
+export NUO_SET_TLS=disable
+$NUODB_HOME/etc/nuoadmin tls $NUO_SET_TLS
+$NUODB_HOME/etc/nuoadmin start
```

Comment:
There is some waiting in the service wrapper script, but you should use the same `nuocmd check servers` command that you used in the Docker Compose test.